pyrcareworld/Test/test_cloth_grasp.py

```diff
@@ -0,0 +1,51 @@
+from pyrcareworld.envs.rcareworld_env import RCareWorld
+
+# Script to test cloth grasping. Paired with "Cloth Grasp" scene.
+if __name__ == "__main__":
+    env = RCareWorld()
+
+    # Create a new cloth representation.
+    cloth = env.create_cloth(id=100, name="Cloth", is_in_scene=True)
+
+    # Create a new robot.
+    robot = env.create_robot(
+        id=315893,
+        # Note: "3158930" is the scene gripper id, but it seems to not work.
```

Comment:
is 315893 working?

Reply:
Yes, 315893 works for how I set up the scene. I placed the gripper script in the same object as the `ControllerAttr`, so this is what seems to have made it work.

Reply:
Ideally the gripper should be detected automatically and assigned an id as 3158930, will look into this issue